hillel/hw20_Classes_Auto+Human/scripts.js

```diff
@@ -0,0 +1,114 @@
+/*ДЗ 20. Створюємо сутності
+
+Створити сутність людини:
+імʼя
+вік
+Метод виведення даних
+Створити сутність автомобіля:
+Характеристики автомобіля окремими властивостями
+Методи:
+Виведення на екран даних про цей автомобіль
+Привласнення цього автомобіля власнику (записати в автомобіль обʼєкт власника)
+
+Що потрібно зробити
+
+1. Створити клас Людина
+    властивості 
+        імʼя
+        вік
+    методи
+        конструктор, який приймає два параметри: імʼя та вік
+        метод, який виводить у консоль інформацію про людину
+2. клас Автомобіль
+    властивості 
+        марка, модель, рік виписку, номерний знак (або на Ваш розсуд)
+        власник
+    методи 
+        конструктор, який приймає чотитри параметри (тобто, всі окрім власника): марка, модель, рік виписку, номерний знак 
+        присвоїти власника - метод повинен приймати екземпляр класу Людина, та зберігати екземпляр класу Людина у відповідному полі, якщо вік більше 18, інакше виводити у консоль відповідне повідомлення
+        метод, який виводить у консоль інформацію про автомобіль та викликає метод виводу інформації класу Людина 
+        для виведення інформації про власника
+
+
+в якості демонстраціїї створити:
+    декілька екземплярів класу Людина
+    декілька екземплярів класу Автомобіль
+    присвоїти власників автомобілям
+    */
+
+class Human {
+    constructor(name,age){
+        this.name=name ;
+        this.age=age;
+    }
+    showHuman(Name,Age){
+        console.log(`  Owner name: ${Name}, age:${Age}`);//this.name this.age
+    }
+}
+
+class Car {
+    constructor(conc, model, year, regNum) {
+        this.concern = conc;
+        this.model = model;
+        this.madeYear = year;
+        this.regNumber = regNum;
+        this.owner = {name:null,age:0}
+
+        
+    }
+    set_owner(human){
```

Comment:
```suggestion
    setOwner(human){
```

у JS ісе ж таки використовується паскаль кейс для назв змінних та методів

Reply:
Так, це недоробка підчас "обкатки"... ++